pkg/nvml/const.go

```diff
@@ -78,18 +78,6 @@ const (
 	VGPU_NAME_BUFFER_SIZE = 64
 	// GRID_LICENSE_FEATURE_MAX_COUNT as defined in nvml/nvml.h
 	GRID_LICENSE_FEATURE_MAX_COUNT = 3
-	// VGPU_SCHEDULER_POLICY_UNKNOWN as defined in nvml/nvml.h
-	VGPU_SCHEDULER_POLICY_UNKNOWN = 0
```

Comment:
Although this is due to the `nvml.h` change, this is breaking from the point of view of consumers -- especially if we consder that we may be loading and older `libnvidia-ml.so` version that still has these defined.

Reply:
NVML only guarantees backwards compatibility within a major release. Are you saying we want to have a stronger guarantee?